### PR TITLE
Cleanups - Technical user

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: cx-portal/src/components/overlays/ServerResponse/index.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ interface ServerResponseOverlayProps {
   intro?: string
   dialogOpen: boolean
   children?: JSX.Element | JSX.Element[]
-  handleCallback?: () => void
+  handleCallback?: any
```
[the remark this answers is not in the document]
Comment (manojava-gk) [reply cx-portal/src/components/overlays/ServerResponse/index.tsx:34]: thats the code smell issue. We can not use arrow functions in the props

[the remark this answers is not in the document]
Comment (nidhigarg-bmw) [reply cx-portal/src/components/overlays/ServerResponse/index.tsx:34]: okay then rest fine for you. Will approve it now

**Review by nidhigarg-bmw — APPROVED** [review]

File: cx-portal/src/components/overlays/ServerResponse/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (manojava-gk) [reply cx-portal/src/components/overlays/ServerResponse/index.tsx]: This change was needed when I replaced the arrow with normal function. Now this is not needed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ n/a
    * stop throbber in case of error and information "no rows" added
    * empty details overlay issue fix by updating the front end business logic of the api key values fetched
 
+* BugFix - Add user role for app not working fix
+* Cleanups
+   * Technical user
```
Comment (jjeroch) [inline CHANGELOG.md:24]: Please move this into the "Unreleased" area, thanks :)

File: cx-portal/src/assets/locales/en/main.json [hunk 3]
```diff
@@ -643,14 +643,18 @@
       "technicalUserHeadline": "Technical User Creation",
       "technicalUserSubheadline": "Create a technical user by adding the relevant information requested below. Please note that the description is used for your own reference only. In case you have a number of technical users it will help you to differenciate between the technical user accounts.",
       "technicalUserSubheadlineSuccess": "Congratulations, the technical user got successfully created. Below you can find all the relevant details needed to apply the technical user for your service connection. Please ensure that you keep this secrets confidential.",
+      "technicalUserSubheadlineError": "Something went wrong. We are looking to it",
```
Comment (jjeroch) [inline cx-portal/src/assets/locales/en/main.json:646]: Lets change this to:
"Something went wrong. The technical user did not got created due to a system error. Please try it later again or contact your administrator."

**Review by jjeroch — CHANGES_REQUESTED** [review]
Small changes only, rest looks good

**Review by oyo — APPROVED** [review]

**Review by oyo — APPROVED** [review]
lgtm

**Review by jjeroch — DISMISSED** [review]

**Comment by jjeroch** [comment]
@oyo I cleaned up the conflict. Could you please approve once again?

**Review by jjeroch — APPROVED** [review]

**Comment by jjeroch** [comment]
> @oyo I cleaned up the conflict. Could you please approve once again?


sorry just released that it isnt needed; i can already merge

**Pull request merged**